### ISSUE TITLE
new `ShapelyDeprecationWarning`

### ISSUE THREAD

**Issue by aaronspring** [regionmask/regionmask#322]
on many `regionmask` calls, e.g. in https://github.com/aaronspring/remote_climate_data/blob/master/notebooks/demo.ipynb

https://github.com/regionmask/regionmask/blob/deaffbe8425f5c7973d34f26bd95d0710c3e7e55/regionmask/core/utils.py#L18

```python
region.plot(label='abbrev', add_land=True, add_ocean=True)

/Users/aaron.spring/anaconda3/envs/remote_climate_data-dev/lib/python3.9/site-packages/regionmask/core/utils.py:18: 
ShapelyDeprecationWarning: __len__ for multi-part geometries is deprecated and will be removed in Shapely 2.0. Check the length of the `geoms` property instead to get the  number of parts of a multi-part geometry.
  polys += list(p)
```

also
```
/Users/aaron.spring/anaconda3/envs/remote_climate_data-dev/lib/python3.9/site-packages/regionmask/core/regions.py:410:
ShapelyDeprecationWarning: Iteration over multi-part geometries is deprecated and will be removed in Shapely 2.0. Use the `geoms` property to access the constituent parts of a multi-part geometry.
  for p in poly:
```

**Comment by mathause** [comment]
Thanks for the report. This should be fixed in master (#291) unless I missed something? Another good reason to get the new version of regionmask out.

**Comment by aaronspring** [comment]
indeed. I was using `0.8.0`. with current `master`, this doesnt happen anymore. but a new release is appreciated.
